wallet/lib/ipc/ipc_engine.dart

```diff
@@ -133,6 +134,7 @@ class IPCEngine {
   }
 
   Future<void> _handleEaselLink(String link) async {
+    navigatorKey.currentState!.popUntil(ModalRoute.withName(RouteUtil.ROUTE_HOME));
```

Comment:
What will happen if the user has no account ?

Reply:
Implemented view NFT with account & bug fixes in `fix/view_nft_with_account`